ddtrace/internal/telemetry/data/payload.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict
+from typing import List
+
+from ....version import get_version
+from .dependency import Dependency
+from .dependency import create_dependency
+from .integration import Integration
+from .metrics import Series
+
+
+class Payload:
+    REQUEST_TYPE = "not-defined"  # type: str
+
+    def to_dict(self):
+        # type: () -> Dict
+        return {}
+
+
+class AppIntegrationsChangedPayload(Payload):
+    REQUEST_TYPE = "app-integrations-changed"  # type: str
+
+    def __init__(self, integrations):
+        # type: (List[Integration]) -> None
+        super().__init__()
+        self.integrations = integrations  # type: List[Integration]
+
+    def to_dict(self):
+        # type: () -> Dict
+        return {"integrations": self.integrations}
+
+
+class AppStartedPayload(Payload):
+    REQUEST_TYPE = "app-started"  # type: str
+
+    def __init__(self, configurations={}, additional_payload={}):
+        # type: (Dict, Dict) -> None
+        super().__init__()
+        self.configuration = configurations  # type: Dict[str, str]
+        self.additional_payload = additional_payload  # type: Dict[str, str]
+        self.dependencies = self.get_dependencies()
+
+    def get_dependencies(self):
+        # type: () -> List[Dependency]
+        import pkg_resources
+
+        dependencies = []
+        for pkg in pkg_resources.working_set:
+            dependency = create_dependency(pkg.project_name, pkg.version)  # type: Dependency
+            dependencies.append(dependency)
+
+        return dependencies
```

Comment:
```suggestion
        return [
            create_dependency(pkg.project_name, pkg.version)
            for pkg in pkg_resources.working_set
        ]
```

alternative approach.